flixel/system/frontEnds/HTML5FrontEnd.hx

```diff
@@ -59,6 +61,49 @@ class HTML5FrontEnd
 	{
 		return Browser.window.innerHeight;
 	}
+	
+	private inline function get_platform():FlxPlatform
+	{
+		
+		if (Browser.navigator.userAgent.indexOf("Win") > -1)
```

Comment:
Oh, and instead of `indexOf()` you can use `.contains()` via `using flixel.util.FlxStringUtil`.